JitsiTrackError.js

```diff
@@ -29,109 +29,111 @@ TRACK_ERROR_TO_MESSAGE_MAP[JitsiTrackErrors.TRACK_MUTE_UNMUTE_IN_PROGRESS]
 TRACK_ERROR_TO_MESSAGE_MAP[JitsiTrackErrors.NO_DATA_FROM_SOURCE]
     = "The track has stopped receiving data from it's source";
 
+
+// FIXME: Using prototype inheritance because otherwise instanceof is not
+// working properly (see https://github.com/babel/babel/issues/3083)
+
 /**
+ *
  * Represents an error that occurred to a JitsiTrack. Can represent various
  * types of errors. For error descriptions (@see JitsiTrackErrors).
  *
  * @extends Error
```

Comment:
It does not extend the Error anymore, is it ?

Do we have any usages of JitsiTrackError that rely on that fact ?